modules/hcp-migrate-aws-single-to-multiarch.adoc

```diff
@@ -0,0 +1,168 @@
+
+.Prerequisites
+
+Before transitioning a hosted cluster from single-architecture to multi-architecture, review the following requirements:
+
+* You have installed an {product-title} management cluster for {aws-short} on {rh-rhacm-first} with the {mce}.
+* You have an existing single-architecture hosted cluster that uses 64-bit AMD variant of the {product-title} release payload.
+* An existing node pool that uses the same 64-bit AMD variant of the {product-title} release payload and is managed by the existing hosted cluster.
+* Ensure that you installed the following command-line tools:
+
+** `oc`
+** `kubectl`
+** `hcp`
+** `skopeo`
+
+.Procedure
+
+. Review the existing {product-title} release image of the single-architecture hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster/<hosted_cluster_name> -o jsonpath='{.spec.release.image}'
+----
++
+.Example output
+[source,terminal]
+----
+quay.io/openshift-release-dev/ocp-release:4.16.5-x86_64
+----
+
+. In your {product-title} release image, if you use the digest instead of the tag, find the multi-architecture tag version of your release image:
+
+.. Get the {product-title} version and save it in the `OCP_VERSION` variable by running the following command:
++
+[source,terminal]
+----
+OCP_VERSION=$(oc image info quay.io/openshift-release-dev/ocp-release@sha256:ac78ebf77f95ab8ff52847ecd22592b545415e1ff6c7ff7f66bf81f158ae4f5e -o jsonpath='{.config.config.Labels["io.openshift.release"]}')
+----
+
+.. Get the multi-architecture tag version of your release image and save it in the `MULTI_ARCH_TAG` variable by running the following command:
++
+[source,terminal]
+----
+MULTI_ARCH_TAG=$(skopeo inspect docker://quay.io/openshift-release-dev/ocp-release@sha256:ac78ebf77f95ab8ff52847ecd22592b545415e1ff6c7ff7f66bf81f158ae4f5e | jq -r '.RepoTags' | sed 's/"//g' | sed 's/,//g' | grep -w "$OCP_VERSION-multi$" | xargs)
+----
+
+.. Save the multi-architecture release image name in the `IMAGE` variable by running the following command:
++
+[source,terminal]
+----
+IMAGE=quay.io/openshift-release-dev/ocp-release:$MULTI_ARCH_TAG
+----
+
+.. Get the list of the multi-architecture image digests, by running the following command:
++
+[source,terminal]
+----
+$ oc image info $IMAGE
+----
++
+.Example output
+[source,terminal]
+----
+OS            DIGEST
+linux/amd64   sha256:b4c7a91802c09a5a748fe19ddd99a8ffab52d8a31db3a081a956a87f22a22ff8
+linux/ppc64le sha256:66fda2ff6bd7704f1ba72be8bfe3e399c323de92262f594f8e482d110ec37388
+linux/s390x   sha256:b1c1072dc639aaa2b50ec99b530012e3ceac19ddc28adcbcdc9643f2dfd14f34
+linux/arm64   sha256:7b046404572ac96202d82b6cb029b421dddd40e88c73bbf35f602ffc13017f21
+----
+
+. Transition the hosted cluster from single-architecture to multi-architecture:
+
+.. Set the `multiArch` flag to `true` on your hosted cluster so that the hosted cluster can create and manage both 64-bit AMD and 64-bit ARM node pools. Run the following command:
```

Comment:
We deprecated this flag in HO in the main branch. It's not used anymore so this section should be removed unless this is for a specific HO version for MCE for instance. 

All you have to do is change the release image now.

Reply:
Neat! Removed this step.